web/src/compose_notifications.ts

```diff
@@ -115,7 +115,10 @@ export function get_muted_narrow(message: Message): string | undefined {
 }
 
 export function get_local_notify_mix_reason(message: Message): string | undefined {
-    assert(message_lists.current !== undefined);
+    if (message_lists.current === undefined) {
+        return undefined;
+    }
```

Comment:
Probably worth a comment for this block noting that for non-message views like inbox, there's no expectation that they see the message after sending it.

Reply:
Done.